docs/Advanced-Usage/FireAxe-Partitioning-onto-Multiple-FPGAs/Running-Exact-Mode-Simulations.rst

```diff
@@ -0,0 +1,83 @@
+Running Exact Mode Simulations
+===================================
+
+In this section, we provide a step-by-step tutorial on how to partition a
+*RocketTile* out from the *SoC* and run exact-mode simulations on EC1 F1.
+Similar steps can be applied to perform locally partitioned FPGA simulations.
+This assumes that you have read the :ref:`FireAxe running fast-mode simulations<FIREAXE-FAST-MODE>`.
+
+1. Building Partitioned Sims : Setting up FireAxe Target configs
+----------------------------------------------------------------
+
+We will be reusing the FireAxe target configurations from :ref:`fast-mode<FIREAXE-FAST-MODE>`.
+
+.. literalinclude:: ../../../sim/firesim-lib/src/main/scala/configs/FireAxeTargetConfigs.scala
+   :language: scala
+   :start-after: DOC include start: F1 Rocket Partition
+   :end-before: DOC include end: F1 Rocket Partition
+
+2. Building Partitioned Sims : `config_build_recipes.yaml`
+----------------------------------------------------------
+
+We can specify the `config_build_recipes.yaml` at this point.
+One thing to note is that we added the `ExactMode_` in the `PLATFORM_CONFIG` field.
+This indicates to the FireAxe compiler to perform additional steps while partitioning so that the target behavior can be simulated in a cycle-exact manner.
+
+.. literalinclude:: ../../../deploy/sample-backup-configs/config_build_recipes.yaml
+   :language: yaml
+   :start-after: DOC include start: F1 Exact Rocket Partition Build Recipe
+   :end-before: DOC include end: F1 Exact Rocket Partition Build Recipe
+
+3. Running Partitioned Simulations : `user_topology.py`
```

Comment:
All references to files that exist committed in the repo should use `gh-file-ref`